coriolis_openstack_utils/actions/coriolis_transfer_actions.py

```diff
@@ -64,6 +62,27 @@ def equivalent_to(self, other_action):
                 return True
         return False
 
+    @abc.abstractmethod
+    def get_transfers_list(self):
+        """Get all coriolis migrations or replicas, in reversed order."""
```

Comment:
1) No referencing any "clild class details" (migration/replica) in parent class.
Please change this to something like "Get all transfer actions of this type."

2) It makes no sense to "add requirements for pointless functionality" such as reversing the returned list to methods when you can just call `reversed()` to the result of the method when you call it: 
https://github.com/cloudbase/coriolis-openstack-utils/pull/23/files#diff-00e607d625ed01dc442cff8e975f0a7bR111